ckanext/security/cache/login.py

```diff
@@ -10,10 +10,21 @@
 
 log = logging.getLogger(__name__)
 
+default_lockout_timeout = 60 * 15
+default_max_login_attempts = 10
+
+
+def lockout_timeout():
+    return int(config.get('ckanext.security.lock_timeout', default_lockout_timeout))
+
+
+def max_login_attempts():
+    return int(config.get('ckanext.security.login_max_count', default_max_login_attempts))
```

Comment:
Since these values can be modified by config, I exposed these functions so that other parts of the code could import them and get the correct values. I didn't end up needing to use them, but I think it's still a better design.